Cython/Compiler/Code.py

```diff
@@ -2718,16 +2718,16 @@ def put_release_ensured_gil(self, variable=None):
             variable = '__pyx_gilstate_save'
         self.putln("__Pyx_PyGILState_Release(%s);" % variable)
 
-    def put_acquire_freethreading_lock(self):
+    def put_acquire_nogil_lock(self):
```

Comment:
Hmm, seeing this now makes it very ambiguous with Cython's `nogil` features. That's even worse because it really gives the wrong context. We use "nogil" in Cython's sources all over the place.

A name like "…_cpython_nogil_…" doesn't seem clear either. Maybe "freethreading" in one word isn't bad after all? `_CPYTHON_NOGIL` for the target macro and `*_freethreading_*` in our code base?

Reply:
Yes, I agree (now you point that out).

I wonder if we should use "freethreading" everywhere (since it's what Python has chosen to call the build). But I'd also be happy to leave the target macro unchanged.

Reply:
Ok. Let's change it everywhere, also in the target macro name. We're freshly introducing it for 3.1, so let's use something really unambiguous.

Reply:
OK - that's at https://github.com/cython/cython/pull/6213